File: PHP/Route.php
<?php

class Route
{
    // Atributes

    private $id;
    private $start;
    private $end;

    // Constructor

    public function __construct($start, $end)
    {
        $this->start = $start;
        $this->end = $end;
    }

    // Getters & Setters

    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getStart()
    {
        return $this->start;
    }

    public function setStart($start)
    {
        $this->start = $start;
    }

    public function getEnd()
    {
        return $this->end;
    }

    public function setEnd($end)
    {
        $this->end = $end;
    }

    // Methods
    
}

File: Java/Route.java
package Java;

import java.util.ArrayList;

public class Route 
{
    private Integer id;
    private ArrayList<Double> start;
    private ArrayList<Double> end;
    
    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }
    public ArrayList<Double> getStart() {
        return start;
    }
    public void setStart(ArrayList<Double> start) {
        this.start = start;
    }
    public ArrayList<Double> getEnd() {
        return end;
    }
    public void setEnd(ArrayList<Double> end) {
        this.end = end;
    }

    
}
